e2e/test/scenarios/dependencies/dependency-graph.cy.spec.ts

```diff
@@ -526,70 +523,14 @@ describe.skip("scenarios > dependencies > dependency graph", () => {
       });
     });
   });
-
-  describe("permissions", () => {
```

Comment:
Can't be tested with the graph being admin only in the UI. Endpoints are tested on the BE.